packages/valory/skills/twitter_scoring_abci/rounds.py

```diff
@@ -607,15 +607,15 @@ class TwitterScoringAbciApp(AbciApp[Event]):
             Event.DONE: TwitterDecisionMakingRound,
             Event.DONE_MAX_RETRIES: TwitterDecisionMakingRound,
             Event.API_ERROR: TwitterMentionsCollectionRound,
-            Event.NO_MAJORITY: TwitterMentionsCollectionRound,
-            Event.ROUND_TIMEOUT: TwitterMentionsCollectionRound,
+            Event.NO_MAJORITY: TwitterRandomnessRound,
+            Event.ROUND_TIMEOUT: TwitterRandomnessRound,
         },
         TwitterHashtagsCollectionRound: {
             Event.DONE: TwitterDecisionMakingRound,
             Event.DONE_MAX_RETRIES: TwitterDecisionMakingRound,
             Event.API_ERROR: TwitterHashtagsCollectionRound,
-            Event.NO_MAJORITY: TwitterHashtagsCollectionRound,
-            Event.ROUND_TIMEOUT: TwitterHashtagsCollectionRound,
+            Event.NO_MAJORITY: TwitterRandomnessRound,
+            Event.ROUND_TIMEOUT: TwitterRandomnessRound,
```

Comment:
We should select new keepers whenever we time out. I found an issue today where the service was stuck in a timeout loop.